### ISSUE TITLE
Bonsai pots refuse to grow if there are 6 currently growing

### ISSUE THREAD

**Issue by salaandora** [thraaawn/BonsaiTrees#100]
**Bonsai Trees version**:bonsai trees 1.1.2 build 144
**Forge version**: 14.23.5.2824
**I am not using Optifine**: False - I am using optifine
**Link to Crashlog**: No crash
**Screenshot (if possible)**: Unneeded

With bonsai trees, I will create multiple bonsai pots in order to harvest multiple trees/types.  However, when using bonsai pots with more than 6 trees, the remaining pots refuse to grow. I am able to put in dirt, and a sapling, into pots past number 6. However, these trees sit at 0.0% and refuse to grow further.

**Steps to reproduce:** In my Minecraft, simply create 7 bonsai pots and plant trees in all of them. The 7th will not grow.

**Comment by thraaawn** [comment]
Obviously not reproducible. Do you really think this has never been tested with more than 6 trees? I've released a mod that is used in multiple modpacks, but I've never tested whether you can grow more than 6 trees? Come on dude, that is a bit insulting.
You deciding whether providing a certain information or not is necessary or not is also not how this bug reporting thing works. Maybe the screenshot shows that you have a block directly above your pot - I don't know. You said **nothing** besides "use the mod as normal, it won't work".

I have no clue what "your" minecraft is. I can't reproduce the issue without "your" minecraft. From a clean slate (only forge + bonsai installed plus a new world) how do I get to the point were the 7th tree won't grow?

tldr: Please provide instructions on how to reproduce this. "In my minecraft" does not help me in any way to see what the issues is here.
